### PR TITLE
new bankin route, I'll use it for bankinnynab 

### PR DESCRIPTION
**Pull request by rienafairefr**
A new route, I intend to use the API at bankin web services/Bridge (rienafairefr/bankinhook repo) in order to fetch transactions and then push it to ynab using fintech-to-ynab

### CODE REVIEW

**Comment by scottrobertson** [comment]
Would you be able to write up some docs on how this works maybe?

**Comment by rienafairefr** [comment]
I've modified the README.md. I'd use fintech-to-ynab as a microservice for the "push to ynab" part, I'll fetch people's transactions from their bank then push it to fintech-to-ynab. Not yet all functional but it's coming.

File: python/routes.py [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 import settings
 import ynab_client
-from functions import create_transaction_from_monzo,create_transaction_from_starling
-
+from functions import create_transaction_from_monzo,create_transaction_from_starling, create_transaction_from_bankin
+#
```
Comment (scottrobertson) [inline python/routes.py]: Do we need the # ?

File: README.md [hunk 2]
```diff
@@ -5,7 +5,10 @@
 
 [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/scottrobertson/fintech-to-ynab)
 
-Automatically push Monzo and Starling transactions into YNAB.
+Automatically push ransactions to YNAB from fintech that can push to webhooks.
```
Comment (scottrobertson) [inline README.md]: Typo here in "transactions"

File: python/routes.py (hunk 1 above)
Comment (rienafairefr) [inline python/routes.py]: nah, it's def a typo

**Comment by scottrobertson** [comment]
Oops forgot this, sorry. Do you mind if i leave this unmerged until it's feature complete?

**Comment by rienafairefr** [comment]
No problem :-) Not sure I will continue pushing on this one actually, still wondering about architecture of all this and where the ynab credentials are stored etc etc. If nothing new in a let's say two weeks, you can close this.

**Pull request closed without merging**